src/custom/state/sentry/updater/index.ts

```diff
@@ -3,44 +3,71 @@ import * as Sentry from '@sentry/browser'
 
 import useIsWindowVisible from 'hooks/useIsWindowVisible'
 import { useSwapState } from 'state/swap/hooks'
-import { SupportedChainId } from 'constants/chains'
 import { useCurrency } from 'hooks/Tokens'
 import { useWalletInfo } from 'hooks/useWalletInfo'
+import { useAppSelector } from 'state/hooks'
+
+function _getSentryChainId(appChainId: null | number, chainId: number | undefined) {
+  const connectedChainId = chainId
```

Comment:
Can we just call parameter `connectedChainId` and omit this line?

Reply:
indeed! i refactored this and never changed the var names because lazy.